### ISSUE TITLE
Storage Box unit & drawer removal broken

### ISSUE THREAD

**Issue by MPZinke** [UTA-FabLab/fabapp#51]
The DB user with permissions to delete a partition for storage box (either all units in a drawer or individual units) was never implemented into Prod & Dev servers.  To fix this, DB user information for user with delete permission specific to storage_box table shall be added to the file ~/class/StorageBox.php

All other features of storage box are functioning, but if a mistake is made (EG getting the partition size wrong), the only way to fix it is to log into the DB and manually remove the errored instance.

**Comment by FabosaurusRex** [comment]
DB user was implemented, we just forgot about it.  PW has been reset, waiting on confirmation its been integrated and ready to test.

**Comment by MPZinke** [comment]
User information has been implemented.  DB user seems to work.
However JS does not work for deleting a unit.  The button for deleting an individual unit remains disabled.  This can currently be negotiated by deleting the entire drawer and recreating the drawer or enabling the button in the dev tools.

**Comment by MPZinke** [comment]
Branch Bugfix-#51 should allow for deletion of non-empty units.  This was done by adjusting the ID variable selected units get with the Drawer object creation—the correct ID should now be assigned to each unit, & the JS evalutation of if a unit is selected works.

**Comment by FabosaurusRex** [comment]
Passed dev server testing, ready for merging w/Dev branch.

**Comment by FabosaurusRex** [comment]
Merged into Dev branch.

**Comment by FabosaurusRex** [comment]
Fix has been rolled out to production.  Closing issue.
